### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by danindugit**
**Describe the pull request:**

I have finished the main header by adding the SOCIS logo on a grid component next to the header message. I made the header responsive so that the columns become rows if the screen becomes small enough (eg. mobile). I animated the header using animate.css.

I added lorem ipsum content for each of the subheaders and updated the titles appropriately. I also added pictures to the first 2 subheaders and made them responsive in a similar fashion to the header grid.
---

Closes: #1 \
Fixes: \
Related: 

---

**Checklist:**

- [x] Run `yarn test` and ensure tests pass
- [x] Run `yarn lint` and ensure linter passes
- [x] Verify the changes work as expected on at least two browsers
- [x] Verify the changes work as expected on mobile view
- [x] Update documentation / not applicable

### CODE REVIEW

File: src/pages/home/HeaderMessage.jsx
Comment (SoorajModi) [inline src/pages/home/HeaderMessage.jsx:15]: While you can do this and works to apply different styles at different widths, a better approach would be to use media queries: https://www.w3schools.com/css/css_rwd_mediaqueries.asp. To use media queries in react Im pretty sure you have to have the css in a separate file and import it, you cant inline it. You could also use [React Responsive](https://www.npmjs.com/package/react-responsive) or [Aphrodite](https://www.npmjs.com/package/aphrodite), but I'll let you decide on what you want to use. Media queries also avoid writing redundant styling, it applies on it on top of the current CSS so you only have to specify the fields you want to change.

I would also change this for the other files where you added logic to a style argument

File: src/pages/home/Subheader.jsx
Comment (SoorajModi) [inline src/pages/home/Subheader.jsx:19]: Also add  `sm={12}` here and wherever else you are using breakpoints

**Review by SoorajModi — CHANGES_REQUESTED** [review]
Okay overall really well done, but just a few details lets get right.

First, I decided to change the style up a little bit so on top of  my comments can you:
- Make sure every filename starts with a lowercase letter, so change `Header.jsx` to `header.jsx`
- Copy and paste the contents of `HomePage.jsx` into `index.jsx` and then delete `HomePage`
- Give `Subheader`, `Subheader2`, and `Subheader3` relevant names that describe what they do

Two related issues
- On desktop view, the buttons arent centered and too far down
- In mobile view Subheader1 currently has the title, description, picture, and then learn more button. Whereas Subheader2 is title, picture, description, then learn more button. Lets make this consistent between both
- I think you should move the `LearnMoreBtn` component into the first grid item and center it. Then on mobile view have the image, then below, the title, description, and learn more button.

**Review by BenPearo — APPROVED** [review]
Looks great. Well done :) This can be merged when/if Sooraj's changes are made

**Review by SoorajModi — APPROVED** [review]

**Pull request merged**
